Tasks/XamarinTestCloud/XamarinTestCloud.ps1

```diff
@@ -58,7 +60,20 @@ if (!$locale)
 {
     throw "Must specify the system language."
 }
-$parameters = "$parameters --locale `"$locale`""
+
+if($locale -eq "user" -and [string]::IsNullOrEmpty($userDefinedLocale))
+{
+    throw "Must specify the system language."
+}
```

Comment:
Instead of checking twice whether $locale -eq "user", can we move the IsNullOrEmpty check/throw into the 2nd if block (below)?